<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StackedWidget</class>
 <widget class="QStackedWidget" name="StackedWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>841</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Yarışma Uygulaması</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="settingsPage">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QPushButton" name="startCompetitionPushButton">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>752</y>
      <width>227</width>
      <height>48</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Yarışmayı Başlat</string>
    </property>
   </widget>
   <widget class="QLabel" name="musicNameLabel">
    <property name="geometry">
     <rect>
      <x>433</x>
      <y>17</y>
      <width>16</width>
      <height>34</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="addCompetitionButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>144</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string>Yarışma Ekle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addQAButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>144</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string>Soru &amp;&amp; Cevap Listesi</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="NextQGroupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>249</width>
      <height>70</height>
     </rect>
    </property>
    <property name="title">
     <string>Bir sorudan diğerine hemen geçmesin</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>İki soru arasında beklenecek süre: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="musicGroupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>134</width>
      <height>99</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>134</width>
      <height>0</height>
     </size>
    </property>
    <property name="title">
     <string>Müzik Olsun mu?</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="musicSelectButton">
         <property name="text">
          <string>Müzik Seç</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="musicLineEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="competitionPage">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="questionLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Comic Sans MS</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="buddy">
         <cstring></cstring>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="continuePausePushButton">
          <property name="text">
           <string>Devam Ettir / Duraklat</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="nextQuestionPushButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Sonraki Soru</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="settingsPushButton">
          <property name="text">
           <string>Ayarlara Dön</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="finishCompetitionPushButton">
          <property name="text">
           <string>Yarışmayı Bitir</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>on_NextQuestionImmediately(bool)</slot>
 </slots>
</ui>
